### DISCUSSION TITLE
Add-PnPTeamsPrivateChannelUser?

### DISCUSSION THREAD

**Discussion by LeonArmston** [pnp/powershell#252] [Ideas]
Hi 

Not sure if there is a PnP command that will help me establish this but I want to add a user to a private channel? 

If there is not then I propose there could be a Add-PnPTeamsPrivateChannelUser? Using these Graph commands for member and owner: https://docs.microsoft.com/en-us/graph/api/channel-post-members?view=graph-rest-1.0&tabs=http

```powershell
Add-PnPTeamsPrivateChannelUser -Team <TeamsTeamPipeBind> -Identity <TeamsChannelPipeBind> -User <String> -Role <String>
```
Thanks

**Comment by uitvoerbaar** [comment]
I am missing this also, and we need this if we want to convert the existing scripts we have.

The old MicrosoftTeams module had a function

`Add-TeamChannelUser -GroupId <String> -DisplayName <String> -User <String> [-Role <String>] [<CommonParameters>]`
